# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/p3nfs/lxhdr_hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./nfsd/nfs_prot.h.orig	2005-08-17 13:11:29.000000000 +0200
+++ ./nfsd/nfs_prot.h	2005-08-17 13:17:27.000000000 +0200
@@ -1,3 +1,14 @@
+
+// glibc defines them. this results in compiler errors
+// in the kernel string.h header...
+#undef strpbrk
+#undef strsep
+#undef strspn
+#undef strcspn
+#undef strcmp
+#undef strncmp
+#undef strchr
+
 #ifdef hpux
 #include <sys/time.h>
 #include <nfs/nfs.h>
